napari/_qt/qt_viewer.py

```diff
@@ -35,6 +36,13 @@
 from .._vispy import create_vispy_visual
 
 
+class OurSceneCanvas(SceneCanvas):
+    def _process_mouse_event(self, event):
+        if event.type == 'mouse_wheel' and len(event.modifiers) > 0:
+            return
+        super()._process_mouse_event(event)
+
```

Comment:
We need to move this to a more appropriate spot - I'm not sure where would be best - qt_utils? It's own class? A new dedicated spot for monkey patching vispy?

Reply:
I think this is an ok spot for it since this is where we import SceneCanvas, but maybe we should rename it KeyFilterSceneCanvas or something descriptive like that? And the class docstring should describe the reason for its existence.

@DragaDoncila note that this is no longer monkey patching, it's subclassing, which is the Right and Good way to do this =), since we won't affect other running libraries using VisPy.